clang/include/clang/Driver/Driver.h

```diff
@@ -658,6 +661,20 @@ class Driver {
   static void getDefaultModuleCachePath(SmallVectorImpl<char> &Result);
 
   bool getOffloadStaticLibSeen() const { return OffloadStaticLibSeen; };
+
+  /// addFPGATempDepFile - Add a file to be added to the bundling step of
+  /// an FPGA object.
+  std::string addFPGATempDepFile(const std::string &Name,
```

Comment:
Does this function need to return Name?

Reply:
Good catch - no, it doesn't given current usage.

Reply:
Cleaned up removed return.